e2e/components/Details.test.ts

```diff
@@ -2,43 +2,56 @@ import {test, expect} from '@playwright/test'
 import {visit} from '../test-helpers/storybook'
 import {themes} from '../test-helpers/themes'
 
-test.describe('Details', () => {
-  test.describe('Default', () => {
-    for (const theme of themes) {
-      test.describe(theme, () => {
-        test('default @vrt', async ({page}) => {
-          await visit(page, {
-            id: 'components-details--default',
-            globals: {
-              colorScheme: theme,
-            },
-          })
+const stories: Array<{title: string; id: string}> = [
```

Comment:
Is that a new practise we're exploring that includes the stories in the e2e test file? 👀

Reply:
Yeah @joshblack started doing this in some of the other tests and thought I would unify around it